index.js

```diff
@@ -1,5 +1,21 @@
 function toRoman(num) {
+   var obj=[ 'M', 'CM', 'D', 'CD', 'C', 'XC', 'L', 'XL', 'X', 'IX', 'V', 'IV', 'I' ]
+   var dec=[ 1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1 ]
+   var res=''
 
+   for (let i = 0; i < obj.length; i++) {
+       var angka=Math.floor(num/dec[i])
+
+        if(angka>0){
+            while(angka>0){
+                res+=obj[i]
+                angka--
+            }
+            num=num%dec[i]
+            toRoman(num)
```

Comment:
Unfortunatelly this is not how recursive work cause even though toRoman function return res but in here, there is no variable for taking the result of toRoman function.
And if I comment line 15, it still work, making it not a recursive function